src/libmongoc/src/mongoc/mongoc-async-cmd.c

```diff
@@ -484,3 +506,39 @@ _mongoc_async_cmd_phase_recv_rpc (mongoc_async_cmd_t *acmd)
 
    return MONGOC_ASYNC_CMD_IN_PROGRESS;
 }
+/*
+ * Copyright 2014-2022 MongoDB, Inc.
```

Comment:
Should we use XXXX-present, or XXXX-<CURRENT_YEAR>?

Reply:
IIRC `XXXX-present` from DRIVERS-409. IMO no need to update in this PR.